Mage.Sets/src/mage/cards/w/WandOfOrcus.java

```diff
@@ -0,0 +1,138 @@
+package mage.cards.w;
+
+import java.util.UUID;
+
+import mage.abilities.Ability;
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.common.AttacksOrBlocksEnchantedTriggeredAbility;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.abilities.effects.common.continuous.GainAbilityAttachedEffect;
+import mage.abilities.effects.common.continuous.GainAbilityControlledEffect;
+import mage.abilities.keyword.DeathtouchAbility;
+import mage.abilities.keyword.EquipAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.AttachmentType;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.constants.Zone;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.game.Game;
+import mage.game.events.DamagedPlayerEvent;
+import mage.game.events.GameEvent;
+import mage.game.permanent.Permanent;
+import mage.game.permanent.token.ZombieToken;
+
+/**
+ *
+ * @author zeffirojoe
+ */
+public final class WandOfOrcus extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("Zombies you control");
+
+    static {
+        filter.add(SubType.ZOMBIE.getPredicate());
+    }
+
+    public WandOfOrcus(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[] { CardType.ARTIFACT }, "{2}{B}");
+
+        this.addSuperType(SuperType.LEGENDARY);
+        this.subtype.add(SubType.EQUIPMENT);
+
+        // Whenever equipped creature attacks or blocks, it and Zombies you control gain
+        // deathtouch until end of turn.
+        Ability deathTouchAbility = new AttacksOrBlocksEnchantedTriggeredAbility(Zone.BATTLEFIELD,
+                new GainAbilityAttachedEffect(DeathtouchAbility.getInstance(), AttachmentType.EQUIPMENT,
+                        Duration.EndOfTurn));
+        deathTouchAbility.addEffect(
+                new GainAbilityControlledEffect(DeathtouchAbility.getInstance(), Duration.EndOfTurn, filter));
+        this.addAbility(deathTouchAbility);
+
+        // Whenever equipped creature deals combat damage to a player, create that many
+        // 2/2 black Zombie creature tokens.
+        this.addAbility(new WandOfOrcusZombieAbility());
```

Comment:
you can use `DealsDamageToAPlayerAttachedTriggeredAbility` instead of creating a custom class

Reply:
Is there an example of this where the damage value is used? That was the reason I created the custom class in the first place. Backfire is the only one i see ?

Reply:
it’s saved to the effects in the `checkTrigger` method

Reply:
makes sense, thanks for the feedback. I committed a re-attempt